src/redprl/redprl.grm

```diff
@@ -256,21 +269,21 @@ customOpParams
   | ([])
 
 metavar
-  : HASH IDENT (IDENT)
+  : HASH ident (ident)
 
 dir
   : param SQUIGGLE_ARROW param ((param1, param2))
 
-term : rawTerm (annotate (Pos.pos (rawTerm1left fileName) (rawTerm1right fileName)) rawTerm)
+term : rawTerm %prec FUN_APP (annotate (Pos.pos (rawTerm1left fileName) (rawTerm1right fileName)) rawTerm)
```

Comment:
There was a reduce/reduce conflict between this rule and the function application rule. This annotation resolved it, essentially by choosing to reduce as many function applications as possible before reducing it to a term.